doc/user/content/release-notes.md

```diff
@@ -107,29 +107,29 @@ These changes are present in [unstable builds](/versions/#unstable-builds) and
 are slated for inclusion in the next stable release. There may be additional
 changes that have not yet been documented.
 
+{{< comment >}}
+Only add new release notes above this line.
+
+The presence of this comment ensures that PRs that are alive across a release
+boundary don't silently merge their release notes into the wrong place.
+{{</ comment >}}
+
+{{% version-header v0.16.0 %}}
+
 - **Breaking change.** Return an error when [`extract`](/sql/functions/extract/)
-  is called with a [`time`] value but a date-related field (e.g., `YEAR`).
+  is called with a [`time`] value but a date-related field (e.g., `YEAR`) {{% gh 9839 %}}.
 
   Previous versions of Materialize would incorrectly return `0` in these cases.
   The new behavior matches PostgreSQL.
 
 - **Breaking change.** Disallow the string `'sNaN'` (in any casing) as a valid
   [`numeric`] value.
```

Comment:
Pre-existing glitch but this is a broken link. Looks like we don't have a [`numeric`] link yet at the bottom.

Reply:
🦅